sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -359,6 +471,50 @@ class RocksDB(
     this
   }
 
+  /**
+   * Initialize in memory values based on the metadata loaded from DFS.
+   * @param metadata: metadata loaded from DFS
```

Comment:
Personally, I don't like comment that provides little information. Either makes it more useful or delete it.

Reply:
done